src/cmds/riichidb.ts

```diff
@@ -151,6 +151,53 @@ export class RiichiDbCommand implements CommandBuilder {
             const gamedata = new AttachmentBuilder(Buffer.from(parse(data[0]), 'utf-8')).setName("DataGame.csv");
             const playerdata = new AttachmentBuilder(Buffer.from(parse(data[1]), 'utf-8')).setName("DataPlayer.csv");
             event.reply({ files: [playerdata, gamedata] });
+        } else if (args[0] === 'verify') {
```

Comment:
i think you could consider moving the verify functionality to a separate function. this way, in the future, we can implement an "auto-verify" after every score insertion.